pxr/usd/usdLux/lightAPI.h

```diff
@@ -359,10 +420,120 @@ class UsdLuxLightAPI : public UsdAPISchemaBase
+    /// 
     /// This makes it easier to independently adjust the power and shape
-    /// of the light, by causing the power to not vary with the area or
-    /// angular size of the light.
+    /// of the light, by causing the total illumination provided by a light to
+    /// not vary with the area or angular size of the light.
+    /// 
+    /// More precisely, this means that the luminance of the light will be
+    /// divided by a factor representing the "size" of the light:
+    /// 
+    /// 
+    /// <center><b>
+    /// L<sub>Scalar</sub> = L<sub>Scalar</sub> / sizeFactor
+    /// </b></center>
+    /// 
+    /// 
+    /// ...where `sizeFactor` = 1 if `normalize` is off, and is calculated
+    /// depending on the family of the light as described below if `normalize`
+    /// is on.
+    /// 
+    /// ### DomeLight / PortalLight:
+    /// 
+    /// For a dome light (and it's henchman, the PortalLight), this attribute is
+    /// ignored:
+    /// 
+    /// <center><b>
+    /// sizeFactor<sub>dome</sub> = 1
+    /// </b></center>
+    /// 
+    /// ### Area Lights:
+    /// 
+    /// For an area light, the `sizeFactor` is the surface area (in world
+    /// space) of the shape of the light, including any scaling applied to the
+    /// light by its transform stack. This includes the boundable light types
+    /// which can have a calculable surface area, such as:
+    /// 
+    /// - MeshLightAPI
+    /// - DiskLight
+    /// - RectLight
+    /// - SphereLight
+    /// - CylinderLight
+    /// - (deprecated) GeometryLight
+    /// 
+    /// <center><b>
+    /// sizeFactor<sub>area</sub> = worldSpaceSurfaceArea(light)
+    /// </b></center>
+    /// 
+    /// ### DistantLight:
+    /// 
+    /// For distant lights, we first define 𝛳<sub>max</sub> as:
+    /// 
+    /// <center><b>
+    /// 𝛳<sub>max</sub> = clamp(toRadians(distantLightAngle) / 2, 0, 𝜋)
+    /// </b></center>
+    /// 
+    /// Then we use the following formula:
+    /// 
+    /// * <i>if 𝛳<sub>max</sub> = 0:</i>
+    /// <center><b>
+    /// sizeFactor<sub>distant</sub> = 1
+    /// </b></center>
+    /// 
+    /// * <i>if 0 < 𝛳<sub>max</sub> ≤ 𝜋 / 2:</i>
+    /// <center><b>
+    /// sizeFactor<sub>distant</sub> = sin²𝛳<sub>max</sub> ⋅ 𝜋
+    /// </b></center>
+    /// 
+    /// * <i>if 𝜋 / 2 < 𝛳<sub>max</sub> ≤ 𝜋:</i>
+    /// <center><b>
+    /// sizeFactor<sub>distant</sub> =
+    /// (2 - sin²𝛳<sub>max</sub>) ⋅ 𝜋
+    /// </b></center>
+    /// 
+    /// This formula is used because it satisfies the following two properties:
+    /// 
+    /// 1. When normalize is enabled, the received illuminance from this light
+    /// on a surface normal to the light's primary direction is held constant
+    /// when angle changes, and the "intensity" property becomes a measure of
+    /// the illuminance, expressed in lux, for a light with 0 exposure.
+    /// 2. If we assume that our distant light is an approximation for a "very
```

Comment:
could we insert a blank line here

Reply:
Done: https://github.com/PixarAnimationStudios/OpenUSD/pull/3182/commits/a1d7d8328149bf6067832d65c79f003fcfc262fa#diff-11634bb5438749e3c529c9f44011d464e79bdb1352c0b7fc7c9408bece4415e4R502

Note that I checked, and this would only affect the in-source comment - the doxygen rendered html stayed the same:

![image](https://github.com/user-attachments/assets/c6a88bfc-3346-445c-89db-52c2eb8b01d7)

If we want to give extra space there, I'll have to... I dunno, add another blank line, or `<p>`, or something...

Reply:
it's fine, just the code